File: add/codesnippet/CSharp/fe018753-fdb7-4d83-aa94-_1.cs
        // Specify client channel properties.
        IDictionary dict = new Hashtable();
        dict["port"] = 9090;
        dict["impersonationLevel"] = "Identify";
        dict["authenticationPolicy"] = "AuthPolicy, Policy";
 
        // Set up a client channel.
        TcpClientChannel clientChannel = new TcpClientChannel(dict, null);
        ChannelServices.RegisterChannel(clientChannel, false);

File: add/codesnippet/CPP/fe018753-fdb7-4d83-aa94-_1.cpp
   // Specify client channel properties.
   IDictionary^ dict = gcnew Hashtable;
   dict[ "port" ] = 9090;
   dict[ "impersonationLevel" ] = "Identify";
   dict[ "authenticationPolicy" ] = "AuthPolicy, Policy";

   // Set up a client channel.
   TcpClientChannel^ clientChannel = gcnew TcpClientChannel( dict, nullptr );
   ChannelServices::RegisterChannel( clientChannel, false );